python/tvm/relay/frontend/pytorch.py

```diff
@@ -2164,6 +2164,24 @@ def is_floating_point(self, inputs, input_types):
         is_float = input_type in ["float32", "float64", "float16", "bfloat16"]
         return _expr.const(is_float)
 
+    def unique(self, inputs, input_types):
+        assert len(inputs) == 4
+        [data, is_sorted, return_inverse, return_counts] = inputs
```

Comment:
Raise an error if `return_inverse` is True, since we don't support it

Reply:
What do you mean? The implementation calculates inverse indices all the time.

Reply:
ah sorry you are right. So in Torchscript `unique` always returns two outputs, regardless of `return_inverse` option? The python one https://pytorch.org/docs/stable/generated/torch.unique.html can return only one output.

Reply:
Yes, I found `aten::_unique2`, the function that `torch.unique` dispatches to when `dim=None`, always return indices. https://github.com/pytorch/pytorch/blob/master/aten/src/ATen/native/Unique.cpp#L69